frontend/amundsen_application/static/js/pages/TableDetailPage/LineageButton/index.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright Contributors to the Amundsen project.
+// SPDX-License-Identifier: Apache-2.0
+
+import * as React from 'react';
+
+import { TableMetadata } from 'interfaces';
+import { isTableLineagePageEnabled } from 'config/config-utils';
+import { logClick } from 'utils/analytics';
+import AppConfig from 'config/config';
+
+export interface LineageButtonProps {
+  tableData: TableMetadata;
+}
+
+const LineageButton: React.FC<LineageButtonProps> = ({
+  tableData,
+}: LineageButtonProps) => {
+  const config = AppConfig.tableLineage;
+  if (!config.isEnabled || !isTableLineagePageEnabled()) return null;
+
+  const { database, cluster, schema, name } = tableData;
+  const href = `/lineage/table/${cluster}/${database}/${schema}/${name}`;
+  if (!href) return null;
+
+  const label = 'Lineage';
+  return (
+    <a
+      className="btn btn-default btn-lg"
+      href={href}
+      role="button"
+      id="table-lineage-button"
+      onClick={logClick}
```

Comment:
We are moving out of this way of tracking events.
We should be using this, plus a redux action to track the event. I see this is reloading the page, but a router's <Link> will give a better experience. Triggering an action will change the global store state and it would be super easy to attach the analytics info to it as we do here: https://github.com/amundsen-io/amundsenfrontendlibrary/blob/master/amundsen_application/static/js/ducks/bookmark/reducer.ts#L29